### DISCUSSION TITLE
Unable to push my code to the "implementation" branch I created

### DISCUSSION THREAD

**Discussion by TheRealKwabena** [DevSkillsHQ/discusssions#21] [Help] [answered]
I am not able to push my code to the cloned repository of the boilerplate for java-springboot. It says that I do not have permission.
![image](https://user-images.githubusercontent.com/101946889/234568327-82d70703-72de-42f9-b22c-0d233261fc0c.png)
How do I fix it? I cloned the repo using SSH.

**Comment by fenske** [comment, marked as answer]
Hi @TheRealKwabena. Anton from Alva is here to help. Per your coding test description, you must push your code to a new branch called 'implementation' on your coding test repository (not the boilerplate repo).

Let me know if you have any other questions.

Best,
Anton.

**Comment by TheRealKwabena** [comment]
Hello,
How do I know which tech stack to use because it was not specified?

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by fenske** [reply]
Unless the hiring team made it explicit, you could use any tech stack you prefer (you could either leverage the ones we support or go with a completely custom setup as described in your README).

**Comment by TheRealKwabena** [comment]
So essentially, I can use the boilerplate code to modify the coding test repository provided.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by fenske** [reply]
Yes
